specification/securityinsights/resource-manager/readme.md

```diff
@@ -26,7 +26,7 @@ These are the global settings for the SecurityInsights API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2021-10
+tag: package-2020-01
```

Comment:
why do we make this change ?

Reply:
I didn't downgrade, I just didn't change the way it's in main. OpenAPIHub automatically changes the default tag but I don't want to change it. I upgraded the basic tag it in another preview version dev branch and I don't want any conflicts when I try to merge this branch to main.
Readme in [main](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/securityinsights/resource-manager/readme.md):
![image](https://user-images.githubusercontent.com/53407600/141077195-400c243e-4de2-4b99-b976-568187db434f.png)

Readme in [2021-09-01-preview dev branch](https://github.com/Azure/azure-rest-api-specs/tree/Microsoft.SecurityInsights-2021-09-01-preview-rebased/specification/securityinsights/resource-manager/readme.md): 
![image](https://user-images.githubusercontent.com/53407600/141077569-431ba003-dac4-4098-850b-670190450634.png)

Reply:
Got it, so you just chenge it back to be consistent with the main branch.